packages/react-components/react-tags/src/components/Tag/useTagStyles.styles.ts

```diff
@@ -0,0 +1,144 @@
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import type { TagSlots, TagState } from './Tag.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
+import { tokens, typographyStyles } from '@fluentui/react-theme';
+
+export const tagClassNames: SlotClassNames<TagSlots> = {
+  root: 'fui-Tag',
+  content: 'fui-Tag__content',
+  media: 'fui-Tag__media',
+  icon: 'fui-Tag__icon',
+  primaryText: 'fui-Tag__primaryText',
+  secondaryText: 'fui-Tag__secondaryText',
+  dismissButton: 'fui-Tag__dismissButton',
+};
+
+/**
+ * Styles for the root slot
+ */
+const useStyles = makeStyles({
+  root: {
```

Comment:
This can be deferred to a later PR but consider using `makeResetStyles()` for the base styles as this greatly reduces the default number of classes on a component which improves style-related perf.

[Example](https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-spinbutton/src/components/SpinButton/useSpinButtonStyles.styles.ts#L22)

Reply:
Agree :) I added a TODO in the comment for future PRs